lib/model/message.dart

```diff
@@ -125,10 +125,21 @@ class MessageStoreImpl with MessageStore {
     if (message == null) return;
 
     message.flags = event.flags;
+    // According to the API, the content of the message is edited if origContent
+    // is not null.
+    if (event.origContent != null) {
+      message.editState = MessageEditState.edited;
```

Comment:
This comment doesn't really say anything that isn't contained in the code just below it; so best to cut the comment.

Concretely, what the code says is: "if origContent is not null, then the message was edited (and not merely moved)". Then "According to the API" doesn't add any information because this function's job is already to implement this part of the API; if this logic is correct, it's because it corresponds to what the API says.

One version of this comment that would add information would be a link to the relevant spot in the API docs. That's helpful because it gives the reader a specific way to verify that the code matches what the API says. If this function is getting to where it feels like it's using some non-obvious details of the API, then probably it'd be good to put such a link at the top. … In fact I see I did that already with the sibling function `_handleUpdateMessageEventMove`; feel free to do something similar for this function.

Reply:
After removing the edit state update nested in the `if (event.renderedContent != null)` block, it seems sufficient to just explain the `origContent` check and link the API doc there. The other ones seem to be straightforward to understand as-is.